### ISSUE TITLE
Bad speedreading https://blog.kagi.com/kagi-orion-public-beta 

### ISSUE THREAD

**Issue by boocmp** [brave/brave-browser#23347]
## Description 

https://blog.kagi.com/kagi-orion-public-beta

![image](https://user-images.githubusercontent.com/73575789/172771503-abc0c94c-3f0f-4889-ba4d-5d0c27f76196.png)

## Steps to Reproduce

   1. 
   2. 
   3. 

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the nightly channel? 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome?

**Comment by LaurenWags** [comment]
Verification in progress with

```
Brave | 1.41.61 Chromium: 103.0.5060.42 (Official Build) nightly (x86_64)
-- | --
Revision | de0d840bf9439c31bd86bf74f065c31fdf9b208d-refs/branch-heads/5060@{#667}
OS | macOS Version 12.4 (Build 21F79)
```

**Comment by MadhaviSeelam** [comment]
Verification `PASSED` using
```
Brave | 1.41.72 Chromium: 103.0.5060.42 (Official Build) beta (64-bit)
-- | --
Revision | de0d840bf9439c31bd86bf74f065c31fdf9b208d-refs/branch-heads/5060@{#667}
OS | Windows 11 Version 21H2 (Build 22000.739)
```
Reproduced in 1.39.122

### Confirmed speedreader version of page is improved with 1.41.72.

1.39.122|1.41.72
---|---
<img width="763" alt="image" src="https://user-images.githubusercontent.com/98358127/174404299-c9fe3d96-e166-496b-a977-306687aafe81.png">|<img width="771" alt="image" src="https://user-images.githubusercontent.com/98358127/174404325-b2add9f1-be03-4138-a62a-b0e87f9656f0.png">

**Comment by stephendonner** [comment]
## Verified `PASSED` using 

Brave | 1.41.80 Chromium: 103.0.5060.53 (Official Build) beta (x86_64)
-- | --
Revision | a1711811edd74ff1cf2150f36ffa3b0dae40b17f-refs/branch-heads/5060@{#853}
OS | macOS Version 11.6.7 (Build 20G630)

### Confirmed the issue with `1.40.105`, and confirmed the fix using `1.41.80`

`1.40.105` | `1.41.80`
-----------|---------
<img width="1312" alt="Screen Shot 2022-06-27 at 2 05 35 PM" src="https://user-images.githubusercontent.com/387249/176035721-40f29bb4-3c6e-44ea-951f-b1b83bf03a9c.png"> | <img width="1380" alt="Screen Shot 2022-06-27 at 2 05 05 PM" src="https://user-images.githubusercontent.com/387249/176035766-f5e96c01-0ab9-4d82-8770-60f16eb5257e.png">

**Comment by stephendonner** [comment]
## Verified `PASSED` using 

Brave | 1.41.80 Chromium: 103.0.5060.53 (Official Build) beta (64-bit)
-- | --
Revision | a1711811edd74ff1cf2150f36ffa3b0dae40b17f-refs/branch-heads/5060@{#853}
OS | Linux

1.40.105 | 1.41.80
-------------|------------
